SmartStorePerformance/android/app/src/main/java/com/salesforce/smartstoreperformance/MainApplication.java

```diff
@@ -0,0 +1,88 @@
+ * are met:
+ * - Redistributions of source code must retain the above copyright notice, this
+ * list of conditions and the following disclaimer.
+ * - Redistributions in binary form must reproduce the above copyright notice,
+ * this list of conditions and the following disclaimer in the documentation
+ * and/or other materials provided with the distribution.
+ * - Neither the name of salesforce.com, inc. nor the names of its contributors
+ * may be used to endorse or promote products derived from this software without
+ * specific prior written permission of salesforce.com, inc.
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+ * ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE
+ * LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+ * CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+ * SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+ * INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+ * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ */
+package com.salesforce.smartstoreperformance;
+
+import android.app.Application;
+
+import com.facebook.react.ReactApplication;
+import com.oblador.vectoricons.VectorIconsPackage;
+import com.facebook.react.ReactNativeHost;
+import com.facebook.react.ReactPackage;
+import com.facebook.react.shell.MainReactPackage;
+import com.salesforce.androidsdk.analytics.security.Encryptor;
+import com.salesforce.androidsdk.app.SalesforceSDKManager;
+import com.salesforce.androidsdk.reactnative.app.SalesforceReactSDKManager;
+
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Application class for our application.
+ */
+public class MainApplication extends Application implements ReactApplication {
+
+	private final ReactNativeHost mReactNativeHost = new ReactNativeHost(this) {
+		@Override
+		public boolean getUseDeveloperSupport() {
+			return BuildConfig.DEBUG;
+		}
+
+		@Override
+		protected List<ReactPackage> getPackages() {
+			return Arrays.asList(
+					new MainReactPackage(),
+            new VectorIconsPackage(),
+					SalesforceReactSDKManager.getInstance().getReactPackage()
+			);
+		}
+
+		@Override
+		protected String getJSMainModuleName() {
+			return "index";
+		}
+	};
+
+	@Override
+	public ReactNativeHost getReactNativeHost() {
+		return mReactNativeHost;
+	}
+
+	@Override
+	public void onCreate() {
+		super.onCreate();
+		SalesforceReactSDKManager.initReactNative(getApplicationContext(), new ReactNativeKeyImpl(),
+                MainActivity.class);
+	}
+}
+
+class ReactNativeKeyImpl implements SalesforceSDKManager.KeyInterface {
```

Comment:
Remove `ReactNativeKeyImpl` - this won't compile with the latest from `dev` anymore since `KeyInterface` doesn't exist anymore.

Reply:
I will update the templates as well.